RetailCoder.VBE/Inspections/QuickFixes/EncapsulateFieldQuickFix.cs

```diff
@@ -32,7 +32,7 @@ public override void Fix()
         {
             var vbe = _target.Project.VBE;
 
-            using (var view = new EncapsulateFieldDialog(_state, _indenter))
+            using (var view = new EncapsulateFieldDialog(new EncapsulateFieldViewModel(_state, _indenter)))
```

Comment:
how do you test the `Fix()` method with a hard dependency on `EncapsulateFieldDialog`?